newm-server/src/main/resources/application.conf

```diff
@@ -265,6 +265,11 @@ mintingNotifications {
         messageUrl = "/email/mint-declined.html"
         messageUrl = ${?EMAIL_MINT_DECLINED_MESSAGE_URL}
     }
+    released {
+         subject = "[IMPORTANT] Your song has been released! \n Preview: Now available on all major streaming platforms"
```

Comment:
The "Preview:" should go nogt go in the subject line. Instead, it should be added inside the .mjml file as an  `<mj-preview>` element. Take a look at any of the existing .mjml file to see how it's done.

Reply:
And btw, don't forget to check-in the .mjml file into the `newm-server/src/main/mjml/email` directory

Reply:
<img width="1035" alt="Screenshot 2024-07-22 at 3 03 39 PM" src="https://github.com/user-attachments/assets/ee3430a3-855d-440b-8947-b47c2fcfb763">